### PR TITLE
Remark: bug fixes

### LINKED ISSUE #191: [PE-D][Tester B] Can unremark entries that already do not have remark
Currently, the unremark function will indicate that it removed a remark despite the entry not having a remark. Maybe can indicate in UG clearer that this is expected behaviour or display a clear message.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: Kb-Tay/ped#1

### LINKED ISSUE #216: [PE-D][Tester B] Remark does not update on Command Detail Panel after editting
When inputting the command `edit 1 re/New remark`, it displays the message as shown in the picture above. However, the view is not updated afterwards.  

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: Kb-Tay/ped#7

### PR DESCRIPTION
**Pull request by alientian**
I believe we cannot add error messages so i did the following for the bugs below:
- Used invalid command error message to resolves #216  
- Update UG description instead of editting the code to resolves #191

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -474,9 +474,15 @@ use the `unremark` command to delete it._
 Directly use the remark command and it overwrites the old remark!
 </div>
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes:**<br>
+* You can unremark a company that already have no remarks. 
+LinkMeIn does not check if a remark exists i.e the same result message will be displayed regardless of whether a remark exist for a company.
```
Comment (tayruxin) [inline docs/UserGuide.md:480]: You can unremark a company that already has no remarks. LinkMeIn does not check for the existence of a remark. The same message will be displayed regardless of whether a remark exists for a company.


Will this phrasing be better

File: docs/UserGuide.md [hunk 2]
```diff
@@ -503,21 +509,22 @@ use the `edit` command to input the accurate details._
 >
 > At least one of the optional fields must be provided.
 
-**Examples:** 
+**Examples:**
 
-1. You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
+**Step 1.** You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
```
Comment (tayruxin) [inline docs/UserGuide.md:514]: You received an email from Microsoft inviting you to interview for a Frontend Developer role, which differs from the Software Engineer role you initially applied for.


will this phrasing be better

**Review by tayruxin — APPROVED** [review]

**Comment by tayruxin** [comment]
LGTM in general but perhaps can consider the phrasing suggested?

File: docs/UserGuide.md (hunk 1 above)
Comment (GlendaChong) [reply docs/UserGuide.md:480]: Maybe specify that the "same message" is the output message

File: docs/UserGuide.md [hunk 3]
```diff
@@ -503,21 +509,22 @@ use the `edit` command to input the accurate details._
...
 
 ![EditDemo1.png](images/edit-command/EditDemo1.png)
-2. The status and role of the 2nd company in the list is edited to be Pending Interview and Frontend Developer respectively.
+
+**Step 2.** The status and role of the 2nd company in the list is edited to be Pending Interview and Frontend Developer respectively.
 
 ![EditDemo2.png](images/edit-command/EditDemo2.png)
 
-3. Go ahead and edit more companies on your own! 
+**Step 3.** Go ahead and edit more companies on your own!
```
Comment (GlendaChong) [inline docs/UserGuide.md:523]: just wondering if this sounds bit funny, like asking them to edit more, but if their input doesn't have any errors, technically they don't have to edit? 

Maybe just say "Some other examples you may try to familiarise with the edit command?" 

Something along the line?

**Review by GlendaChong — APPROVED** [review]

**Pull request merged**